client/state/data-getters/request-geo-location.js

```diff
@@ -1,14 +0,0 @@
-import { requestHttpData } from 'calypso/state/data-layer/http-data';
-import { http as rawHttp } from 'calypso/state/http/actions';
-
-const requestGeoLocation = () =>
```

Comment:
This is still included in the `httpData` example: https://github.com/Automattic/wp-calypso/blob/migrate/request-geolocation/client/state/data-layer/http-data.ts#L249, should we replace it with another one before 🚢 ?

Reply:
We're going to remove `http-data`, hopefully soon, and no new usages should be being added. That's why I didn't want to spend time inventing another example.